multi-node/vagrant/kubeconfig

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Config
+clusters:
+- cluster:
+    certificate-authority: ssl/ca.pem
+    server: https://172.17.4.101:443
+  name: vagrant-cluster
```

Comment:
Just for consistency sake, can we keep same names as given in the local-user examples. In this case "vagrant-multi"